### ISSUE TITLE
Rotate in X,Y axis

### ISSUE THREAD

**Issue by F-Burning** [opentoonz/opentoonz#4624]
### Describe the solution you'd like

Basically this:

![k](https://user-images.githubusercontent.com/74098734/204841437-82aecf1a-30b3-4f5e-9175-9294bf1ede31.jpg)

**Comment by flurick** [comment]
It is not obvious but you can rotate the view with the rotate tool. Or is this about animating a layer in 3D?

**Comment by F-Burning** [comment]
Yes, I mean a 3D environment, but searching the internet, i found another thread where rodney explains why it's not viable, since Opentoonz uses tricks to simulate depth, but it's still 2D.

So, I should have been well informed before launching the suggestion, sorry.

**Comment by RodneyBaker** [comment]
As a follow up to this...

I'm confident Opentoonz will eventually have a 'real' 3D viewing space but that is a major undertaking that likely will not happen soon.  In the interim there are many ways to get at the same thing and most rely on use of third party applications. 
Don't look for a fully tooled 3D View until v3.0 however.  ;)

 There are too many programs to list but here are a few that easily plug into the Opentoonz pipeline.
- Windows Paint 3D (comes with every release of WIndows and allows rotation of images and models in 3D space) Linux and Mac likely have equivalent programs
- Davinci Resolve (specifically the Fusion page which has enormous 3D node variations).  Free version but also studio version which includes a standalone Fusion Studio.
- Blender (probably the defacto 3D environment for most.  Well worth investigating especially if new to 3D as the rule of 'first learned best remembered' applies to this very complex program.
- Many others both free and paid (Godot, Unreal, Unity, Animation:Master, etc.)
-

**Comment by RodneyBaker** [comment]
Tranferring this request to Opentoonz-docs for general purpose roadmapping...
